EasyCommands/BlockHandlers/BlockHandlers.cs

```diff
@@ -315,9 +324,11 @@ public abstract class BlockHandler<T> : BlockHandler where T : class {
             PropertyHandler<T> TypedPropertyHandler<U>(GetTypedProperty<T, U> Get, SetTypedProperty<T, U> Set, Func<Primitive, U> Cast, U delta) => new SimpleTypedHandler<T, U>(Get, Set ?? ((b, v) => { }), Cast, delta);
         }
 
-        public class TypeHandler<T, U> {
-            public PropertyHandler<T> handler;
-            public U[] supportedTypes;
+        public abstract class MultiInstanceBlockHandler<T> : BlockHandler<T> where T : class {
+            public override IEnumerable<T> SelectBlocksByType<U>(List<U> blocks, Func<U, bool> selector = null) =>
+                blocks?.Where(b => selector?.Invoke(b) ?? true).SelectMany(b => GetInstances(b)) ?? Enumerable.Empty<T>();
```

Comment:
I wonder if we should add a List<T>(IEnumerable<T> things) utility.  At some point it may be less chars. "IEnumerable" vs "List", especially given NewList<T>(), causes chars to add up for sure.  7 chars diff per Enumerable vs List.

Reply:
you need to count the more or less mandatory `.ToList()` when using lists aswell which ats 9 more chars.
The advantage of IEnumerable is that the traversal is deferred until it is realized (e.g. evtl calling to list)